CMakeLists.txt

```diff
@@ -584,6 +584,9 @@ endif()
 option(WITH_SEASTAR "Build seastar components")
 set(HAVE_SEASTAR ${WITH_SEASTAR})
 
+option(WITH_SYSTEM_NPM "Assume that npm is already installed through packages" OFF)
+set(HAVE_SYSTEM_NPM ${WITH_SYSTEM_NPM})
```

Comment:
please
```cmake
if(WITH_SYSTEM_NPM)
  find_program(NPM npm)
  if(NOT NPM)
    message(FATAL_ERROR "Can't find npm.")
  endif()
endif()
```
and replace `npm` with `${NPM}` in `add_custom_command()` calls calling `npm`.

no need to define `HAVE_SYSTEM_NPM`. `HAVE_FOO` is normally used by source code.

Reply:
@tchaikov 
Will fix.